scalameter-core/src/main/scala/org/scalameter/package.scala

```diff
@@ -19,6 +19,18 @@ package object scalameter extends MeasureBuilder[Unit, Double](
   MeasureBuilder.average
 ) {
 
+  implicit def inline[T: Ordering](measurer: Measurer[T]): MeasureBuilder[Unit, T] = {
```

Comment:
I've added it to enable inline benches for all kind of measurers. I suppose that the alternative could be to make `MeasureBuilder.withMeasurer` generic, but that would require to either change default aggregator to median or enforce changing of measurer and aggregator in one method.